### ISSUE TITLE
Helper not defined: \"this.\"

### ISSUE THREAD

**Issue by azerupi** [sunng87/handlebars-rust#100]
So I'm trying to access `this.[0].issue_number` in a partial but I get the following render error:

```
Error was: RenderError(RenderError { desc: "Helper not defined: \"this.\"", template_name: Some("index"), line_no: Some(3), column_no: Some(28) })
```

The thing is that I can access that value with log: `{{log this.[0].issue_number}}` but not with `{{this.[0].issue_number}}`. 

I don't know handlebars enough but this seems like a bug, no?

**Comment by sunng87** [comment]
It seems to be a parser issue. Let me check.

**Comment by sunng87** [comment]
hi @azerupi , this should be fixed in 0.20.5.

**Comment by azerupi** [comment]
Thanks! :)
